tests/user/user_result_samples/alter_user_result_error_uid_secret.json

```diff
@@ -0,0 +1,27 @@
+{
+  "securityResult": {
+    "user": {
+      "name": "SQUIDWRD",
+      "operation": "set",
+      "requestId": "UserRequest",
+      "info": [
+        "Definition exists. Add command skipped due  to precheck option"
+      ],
+      "commands": [
+        {
+          "safReturnCode": 8,
+          "returnCode": 16,
+          "reasonCode": 8,
+          "image": "ALTUSER SQUIDWRD  NOSPECIAL      OMVS     (HOME        ('/u/clarinet') NOPROGRAM      UID         (********))",
+          "messages": [
+            "IKJ56702I INVALID UID, 90000000000",
```

Comment:
This seems like a problem. We might want to mark this functionality as experimental for now, since we can't really guarantee that custom secret traits will be fully redacted using the current secrets redaction logic.

Reply:
I agree that marking it experimental with this explicit caveat is appropriate. We can discuss how best to handle this in a future update. I do have a few ideas already.